src/main/java/CinemaRoomRestService/RestService/api/controller/CinemaRoomController.java

```diff
@@ -0,0 +1,47 @@
+package CinemaRoomRestService.RestService.api.controller;
+
+import CinemaRoomRestService.RestService.api.dto.StatisticsDTO;
+import CinemaRoomRestService.RestService.api.dto.CinemaRoomDTO;
+import CinemaRoomRestService.RestService.api.dto.SeatDTO;
+import CinemaRoomRestService.RestService.api.dto.TokenOfSeatDTO;
+import CinemaRoomRestService.RestService.api.mapper.SeatMapper;
+import CinemaRoomRestService.RestService.service.BookingService;
+import CinemaRoomRestService.RestService.domain.entity.Seat;
+import CinemaRoomRestService.RestService.domain.entity.TokenOfSeat;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+public class CinemaRoomController {
+    private final BookingService bookingService;
+    @Autowired
+    SeatMapper seatMapper;
+
+    @Autowired
+    public CinemaRoomController(BookingService bookingService) {
+        this.bookingService = bookingService;
+    }
+
+    @GetMapping("/seats")
+    public CinemaRoomDTO getSeatInfo() {
+        return seatMapper.toCinemaRoomDTO(bookingService.getSeatInfo());
+    }
+
+    @PostMapping("/purchase")
+    public @ResponseBody
+    TokenOfSeatDTO bookingOfSeat(@RequestBody Seat reqSeat) {
```

Comment:
Your Controller should use SeatDTO not Domain Seat, create one more DTO and name it BookCommand, (You can have different DTOs types  (DTO, Filter, Command))

Reply:
Fixed